Economy.md

```diff
@@ -0,0 +1,93 @@
+```yml
+# Defines the balance with which new players begin. Defaults to 0.
+starting-balance: 0
+
+# Defines the cost to use the given commands PER USE.
+# Some commands like /repair have sub-costs, check the wiki for more information.
+command-costs:
+  # /example costs $1000 PER USE
+  #example: 1000
+  # /kit tools costs $1500 PER USE
+  #kit-tools: 1500
+
+# Set this to a currency symbol you want to use.
+# Remember, if you want to use special characters in this document, 
+# such as accented letters, you MUST save the file as UTF-8, not ANSI.
+currency-symbol: '$'
+
+# Enable this to make the currency symbol appear at the end of the amount rather than at the start.
+# For example, the euro symbol typically appears after the current amount.
+currency-symbol-suffix: false
+
+# Set the maximum amount of money a player can have.
+# The amount is always limited to 10 trillion because of the limitations of a java double.
+max-money: 10000000000000
+
+# Set the minimum amount of money a player can have (must be above the negative of max-money).
+# Setting this to 0, will disable overdrafts/loans completely.  Users need 'essentials.eco.loan' perm to go below 0.
+min-money: -10000
+
+# Enable this to log all interactions with trade/buy/sell signs and sell command.
+economy-log-enabled: false
+
+# Enable this to also log all transactions from other plugins through Vault.
+# This can cause the economy log to fill up quickly so should only be enabled for testing purposes!
+economy-log-update-enabled: false
+```
+EssentialsX adds the ability to specify the minimum transaction amount to prevent the dreaded microtransactions:
+```yml
+# Minimum acceptable amount to be used in /pay.
+minimum-pay-amount: 0.001
+```
+EssentialsX adds currency format to improve the localization. You may uncomment the 'currency-symbol-format-locale' to use our preformatted styles or uncomment 'currency-format' and create your own:
+```yml
+# The format of currency, excluding symbols. See currency-symbol-format-locale for symbol configuration.
+#
+# "#,##0.00" is how the majority of countries display currency.
+#currency-format: "#,##0.00"
+
+# Format currency symbols. Some locales use , and . interchangeably.
+# Some formats do not display properly in-game due to faulty Minecraft font rendering.
+#
+# For 1.234,50 use de-DE
+# For 1,234.50 use en-US
+# For 1'234,50 use fr-ch
+#currency-symbol-format-locale: en-US
+```
+#### Permissions
+[Permissions for commands can be found here](https://essinfo.xeya.me/permissions.html) (Community maintained list)
+
+Important permissions include `essentials.eco` and  `essentials.pay`.
+## Signs & Shops
+Signs may be used to create shops where players can buy and sell items/blocks to and from the server in exchange for money. 
+#### Signs
+To create a sign, ensure that you have the `signs.create.trade` permission and that you have uncommented `- buy` and `- sell` under `enabledSigns:` in the config.yml
```

Comment:
The `essentials.signs.<action>.trade` permissions are for the `[trade]` sign, which this section doesn't actually explain. The `buy` and `sell` permissions are for the `[buy]` and `[sell]` signs respectively. This also doesn't explain that players need to be given permissions to *use* the signs too, or that creating `[buy]` and `[sell]` should be limited to admins only.